superset-frontend/src/components/Menu/Menu.jsx

```diff
@@ -51,11 +52,19 @@ const propTypes = {
   }).isRequired,
 };
 
+const Styles = styled.header`
```

Comment:
Maybe a name like `StyledHeader` might be more clear than `Styles`?

Reply:
I think it's fine to use `Styles`. This is for internal use, kind of like the `<style>` tag in Vue's [Single File Component](https://vuejs.org/v2/guide/single-file-components.html).